docs/Customization/Memory-Hierarchy.rst

```diff
@@ -62,22 +73,26 @@ Note that these configurations fully remove the L2 cache and mbus.
 
 .. code-block:: scala
 
-    class SmallRocketConfigNoL2 extends Config(
-        new WithTSI ++
-        new WithNoGPIO ++
-        new WithBootROM ++
-        new WithUART ++
-        new freechips.rocketchip.subsystem.WithNoMMIOPort ++
-        new freechips.rocketchip.subsystem.WithNoSlavePort ++
-        new freechips.rocketchip.subsystem.WithNSmallCores(1) ++
-        new freechips.rocketchip.system.BaseConfig)
-
-    class ScratchpadRocketConfig extends Config(
-        new freechips.rocketchip.subsystem.WithNoMemPort ++
-        new freechips.rocketchip.subsystem.WithNMemoryChannels(0) ++
-        new freechips.rocketchip.subsystem.WithNBanks(0) ++
-        new freechips.rocketchip.subsystem.WithScratchpadsOnly ++
-        new SmallRocketConfigNoL2)
+    class ScratchpadSmallRocketConfig extends Config(
```

Comment:
Obligatory code-in-docs comment: This makes my eyes bleed.

Reply:
I moved some of the configs in this section into RocketConfigs, and reduced the rest to just show the mixin